packages/http/libraries/core/src/models/RequestMethodType.ts

```diff
@@ -0,0 +1,10 @@
+export enum RequestMethodType {
+  GET = 'get',
+  POST = 'post',
+  PUT = 'put',
+  DELETE = 'delete',
+  PATCH = 'patch',
+  OPTIONS = 'options',
+  //HEAD = 'head',
```

Comment:
Let's include head requests as well

Reply:
I have to find a way to include head function to hono driver. Anyways, I include it.

Reply:
Yeah, If the adapter cannot implement a feature we can trow a custom error

Reply:
HEAD value added